pkgs/racket-doc/scribblings/guide/performance.scrbl

```diff
@@ -577,8 +577,8 @@ In general, Racket re-uses the storage for a value when the
 garbage collector can prove that the object is unreachable from
 any other (reachable) value. Reachability is a low-level, 
 abstraction breaking concept (and thus one must understand many
-details of the runtime system's implementation to accurate predicate
-precisely when values are reachable from each other),
+details of the runtime system's implementation to accurately predict
```

Comment:
```suggestion
details of the runtime system's implementation to accurately predict precisely
```

Reply:
Hi Sam,

I thought about this, too. However, in my opinion "accurate" and "precise" are so close in meaning that nothing is won if you use both words; it looks rather redundant. (When deciding which of "accurately" or "precisely" to remove, I stuck with "accurate" because it sounds better with "predict.")

I can accept the suggestion if you want me to :-) , but I think keeping the sentence simple(r) would be better for most readers.

Reply:
Updated the paragraph after discussion with Sam. I tried to express the gist of the paragraph, intending to make it a bit easier to read.

Reply:
Not sure if @samth wants to include the word "precise" because it's a technical word ("precise GC"). Though precise GC is also known as accurate/exact GC...

Reply:
Oh... I'm two minutes too late. Looks like you talked to each other already...